### DISCUSSION TITLE
dissimilarity.py missing from methods folder

### DISCUSSION THREAD

**Discussion by JackyZzZz** [theochem/Selector#207] [Q&A] [answered]
I noticed that we need to import ```dissimilarity.py``` from ```methods``` in the ```Running Dissimilarity Algorithms``` example in the documentation. However, it seems like this file is missing from the methods folder.

P.S: I think it is renamed to distance.py, so we should probably update the documentation.

## Add-on:
- the first parameter of ```select``` function in base.py is ```X```, not ```arr``` in the documentation

**Comment by FanwangM** [comment, marked as answer]
Thank for noticing this.
I will do a cleaning up after May 31st, and we can probably talk about it in our next meeting.  @JackyZzZz
